vars/sonarExecuteScan.groovy

```diff
@@ -14,7 +14,7 @@ import java.nio.charset.StandardCharsets
 @Field Set GENERAL_CONFIG_KEYS = [
     /**
      * Pull-Request voting only:
-     * The URL to the Github API. see https://docs.sonarqube.org/display/PLUG/GitHub+Plugin#GitHubPlugin-Usage
+     * The URL to the Github API. see [https://docs.sonarqube.org/display/PLUG/GitHub+Plugin#GitHubPlugin-Usage](https://docs.sonarqube.org/display/PLUG/GitHub+Plugin#GitHubPlugin-Usage)
```

Comment:
With the suggested change the page looks like "Github API. see https://docs.sonarqube.org/display/PLUG/GitHub+Plugin#GitHubPlugin-Usage", the url is a link pointing to itself. Looks from my point of view not that nice. I'm not strongly against this approach, but would't it be better to have something like "Github API. see here", where "here" is a link?

To be more precise: what I suggest is

```
<...> Github API. see [here](https://docs.sonarqube.org/display/PLUG/GitHub+Plugin#GitHubPlugin-Usage)
```
Applies also for the next change below